docs/resources/logbook/2023_lhc.md

```diff
@@ -10,28 +10,41 @@ The tables below can be sorted by clicking next to the column headers.
+| 2023-04-08 | Commissioning |    NL Studies at Injection            |   [Shift Plan][nl_studies_injection]{target=\_blank .cern_login} /  [Summary][nl_studies_injection_sum]{target=\_blank .cern_login}   |
+| 2023-04-10 | Commissioning |    Ions Virgin + Local Coupling       |   [Entry][ions_virgin_coupl]{target=\_blank .cern_login}                                                                              |
+| 2023-04-16 | Commissioning |    Ions Global Corr and Validation    |   [Shift Plan][nl_studies_injection]{target=\_blank .cern_login} / [Summary][nl_studies_injection_sum]{target=\_blank .cern_login}    |
+
+
 
 
 <!-- All the links below -->
 [inj_linear_optics]:                        https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-03-28T00%3A00%3A00&dateTo=2023-03-28T23%3A59%3A59&eventToHighlight=3739094 
 [inj_linear_optics_sum]:                    https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-03-29T00%3A00%3A00&dateTo=2023-03-29T23%3A59%3A59&eventToHighlight=3739271 
 [squeezed_linear_optics]:                   https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-01T00%3A00%3A00&dateTo=2023-04-01T23%3A59%3A59&eventToHighlight=3741499 
 [squeezed_linear_optics_sum]:               https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-01T00%3A00%3A00&dateTo=2023-04-01T23%3A59%3A59&eventToHighlight=3741722
+[meas_in_ramp]:                             https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-06T00%3A00%3A00&dateTo=2023-04-06T23%3A59%3A59&eventToHighlight=3744741 
+[meas_in_ramp_sum]:                         https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-06T00%3A00%3A00&dateTo=2023-04-06T23%3A59%3A59&eventToHighlight=3744954 
 [a4_corrections_30cm]:                      https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745378
 [a4_corrections_30cm_sum]:                  https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745433
 [nl_studies_injection]:                     https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745829
 [nl_studies_injection_sum]:                 https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-08T23%3A00%3A00&dateTo=2023-04-09T07%3A00%3A00&eventToHighlight=3746004
+[ions_virgin_coupl]:                        https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-10T00%3A00%3A00&dateTo=2023-04-10T23%3A59%3A59&eventToHighlight=3746402
```

Comment:
This link directs to a logbook entry with measurements of the kickgroup. Is there not a better one?

Reply:
No, that shift was not well documented